src/components/TextField/TextField.tsx

```diff
@@ -37,11 +40,26 @@ class TextField extends React.Component<TextFieldProps & DebouncedOnChangePrivat
         suffix={this.props.suffix}
         underlined={this.props.underlined}
         onChanged={this.props.unifiedOnChange}
+        componentRef={this.getRef}
         {...getBaseTextFieldProps(this.props)}
       />
     );
   }
 
+  public focus = () => {
```

Comment:
do we need to also duplicate these methods and ref on `TextArea`? we may want to implement an internal `BaseTextField` component which restricts the Fabric props to only what our `TextField` and `TextArea` need (and handle CSS styling), then our `TextField` and `TextArea` components would be super thin wrappers which just manage their list of props

Reply:
I don't have a plan to enable this on TextArea.  Would you be comfortable if we deferred this until needed?

Reply:
yep that's fine